deploy/grafana/dashboards/clusters.json

```diff
@@ -3717,17 +3717,17 @@
         "current": {},
         "datasource": {
           "type": "prometheus",
-          "uid": "${datasource}"
+          "uid": "${DS_PROMETHEUS}"
         },
-        "definition": "label_values(kube_node_info{cluster=\"$cluster\"},node)",
+        "definition": "label_values(kube_node_info{cluster=\"$cluster\"},internal_ip)",
         "hide": 0,
         "includeAll": true,
         "label": "Nodes",
         "multi": true,
         "name": "Nodes",
         "options": [],
         "query": {
-          "query": "label_values(kube_node_info{cluster=\"$cluster\"},node)",
+          "query": "label_values(kube_node_info{cluster=\"$cluster\"},internal_ip)",
```

Comment:
I think we could use named capture groups here to separate displayed value and used value, e.g.
``` json
       "query": {
          "qryType": 3,
          "query": "query_result(kube_node_info)",
          "refId": "PrometheusVariableQueryEditor-VariableQuery"
        },
        "refresh": 2,
        "regex": "/node=\"(?<text>[^\"]+)|internal_ip=\"(?<value>[^\"]+)/g",
```

This would allow users to select nodes based on names but filter panels by underlying IPs